src/test/integration/java/com/commercetools/payment/handler/CommercetoolsCreatePaymentInstallmentIT.java

```diff
@@ -66,6 +67,12 @@ public void setUp() {
         super.setUp();
     }
 
+    @After
+    @Override
+    public void tearDown() {
+        super.tearDown();
```

Comment:
Why it was not required before?

Reply:
Probably because when running the tests, it was pointed towards the paypal_plus staging CT project which already had the expected product types. Therefore it was not required to create new product types in CT project. Therefore no need of clearing the old resources in CT project.